db/version_edit_handler.cc

```diff
@@ -874,12 +947,13 @@ Status VersionEditHandlerPointInTime::MaybeCreateVersion(
   // A new version will created if:
```

Comment:
`A new version will be created if:`

since we are touching the area, we can fix the typo :)